MetaMask/metamask-mobile
# feat: 2794 account edit permission multichain bottomsheet UI changes only

## **Description**
Modifies the edit account screen


## **Related issues**

Contributes to resolve issue: https://github.com/MetaMask/MetaMask-planning/issues/2794

## **Manual testing steps**

1. Connect one account to a dapp in the in app browser
2. Select the dapp, in the header click the account avatar, a bottom sheet comes up, click connect more accounts
3. You should see the updated edit accounts screen

## **Screenshots/Recordings**

| Before       | After        |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/5bb7829d-31b2-42fb-afc9-0c5be06d3651"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/d3d76050-330c-4c79-8477-9b42b17ee7ed"> |



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
